src/main/java/twilightforest/biomes/RegistryBiomeEvent.java

```diff
@@ -12,7 +12,6 @@
 
 @EventBusSubscriber
 public final class RegistryBiomeEvent {
-	@SuppressWarnings("OverlyCoupledMethod")
```

Comment:
all of these suppresswarnings you're removing that "seemingly do nothing", actually do indeed do things on some IDE's.
https://gist.github.com/vegaasen/157fbc6dce8545b7f12c is one list of them